mlflow/deployments/cli.py

```diff
@@ -0,0 +1,103 @@
+import click
+from mlflow.utils import cli_args
+from mlflow.deployments import interface
+from mlflow.deployments.utils import parse_custom_arguments
+
+
+deployment_target = click.option("--target", "-t", required=True, help="Deployment target")
+deployment_id = click.option("--id", "_deployment_id", required=True,
+                             help="Deployment ID for the deployment that needs to be deleted")
+context_settings = dict(allow_extra_args=True, ignore_unknown_options=True,)
+
+
+@click.group("deployments")
+def commands():
+    """
+    Deploy MLflow models.  Downstream functions calls the plugin registered for the
```

Comment:
```suggestion
    [experimental] Deploy MLflow models to custom targets.

    Downstream functions calls the plugin registered for the...
```

The idea is to make the `deployments` commands stand out as different (and experimental) when users run `mlflow --help`, which currently yields the following (note that the `models` & `deployments` CLIs sound pretty similar):

```
Usage: mlflow [OPTIONS] COMMAND [ARGS]...

Options:
  --version  Show the version and exit.
  --help     Show this message and exit.

Commands:
  artifacts    Upload, list, and download artifacts from an MLflow artifact...
  azureml      Serve models on Azure ML.
  db           Commands for managing an MLflow tracking database.
  deployments  Deploy MLflow models.
  experiments  Manage experiments.
  gc           ..
  models       Deploy MLflow models locally.
  run          Run an MLflow project from the given URI.
  runs         Manage runs.
  sagemaker    Serve models on SageMaker.
  server       Run the MLflow tracking server.
  ui           Launch the MLflow tracking UI for local viewing of run...
```

but would ideally yield something like:

```
Usage: mlflow [OPTIONS] COMMAND [ARGS]...

Options:
  --version  Show the version and exit.
  --help     Show this message and exit.

Commands:
  artifacts    Upload, list, and download artifacts from an MLflow artifact...
  azureml      Serve models on Azure ML.
  db           Commands for managing an MLflow tracking database.
  deployments  [experimental] Deploy MLflow models to custom targets.
  experiments  Manage experiments.
  gc           ..
  models       Deploy MLflow models locally.
  run          Run an MLflow project from the given URI.
  runs         Manage runs.
  sagemaker    Serve models on SageMaker.
  server       Run the MLflow tracking server.
  ui           Launch the MLflow tracking UI for local viewing of run...
```